docs/en/developer.md

```diff
@@ -423,6 +423,38 @@ for the advanced settings.
 
 ## Advanced features
 
+### Allowing email login on LDAP login form
+
+If you have registered `LDAPAuthenticator` (see "Configure SilverStripe Authenticators" section), you may notice the only
+accepted login is username on the form. This may be confusing if your users are more familiar logging in with using their email
+address.
+
+To allow logging in with email in addition to username, set the following configuration:
+
+```yaml
+LDAPAuthenticator:
+  allow_email_login: 'yes'
+```
+
+Note that your LDAP users logging in must have the `mail` attribute set, otherwise this will not work.
+
+### Falling back authentication on LDAP login form
+
+You can allow users who have not been migrated to LDAP to authenticate via the default `MemberAuthenticator`
```

Comment:
I'd somehow mention that this is different to loading multiple authenticators and having them show under multiple tabs - maybe say the fallback is transparent to the user?